wrappers/python/tests/systems/alanine-dipeptide-amoeba-forces.xml

```diff
@@ -1,32 +1,32 @@
-		<Force x="-518.9062723002917" y="686.3850843706331" z=".7311137123231983"/>
-		<Force x="-105.62094199225953" y="710.9845370176521" z="-508.0677463920413"/>
-		<Force x="285.6690654282444" y="-111.35548608152155" z="847.8096733172159"/>
-		<Force x="-396.4822698774795" y="24.462223992557014" z="297.9581483103431"/>
-		<Force x="-168.22340166944585" y="110.99779797739818" z="-824.0916382722021"/>
-		<Force x="805.7023041833917" y="15.738288953519556" z="-340.2263402925698"/>
-		<Force x="366.3025998383222" y="-647.7535895818237" z="-273.83047817845636"/>
-		<Force x="-649.7310636705703" y="-495.52797252280067" z="317.36013856851315"/>
-		<Force x="-480.120788018345" y="172.44477988154952" z="-16.235455767965004"/>
-		<Force x="-486.20230228601736" y="982.6376366390087" z="127.20818643942287"/>
-		<Force x="820.5730252078641" y="-78.71234329038225" z="109.31395566365498"/>
-		<Force x="355.03388135028814" y="-1391.2776654673507" z=".9942543092954484"/>
-		<Force x="-511.3467906691061" y="677.7717086552518" z=".008253291392112616"/>
-		<Force x="293.66157422204157" y="452.68343054060256" z="502.83463917630775"/>
-		<Force x="293.5683952568295" y="452.70832957117113" z="-502.49604726620026"/>
+		<Force x="-36.63343579012365" y="-752.262766500235" z="-.38948064609111555"/>
+		<Force x="1213.0199051904176" y="639.092807771111" z="-3.635766271628655"/>
+		<Force x="-474.1729469409308" y="176.9414396833211" z="597.5160403003063"/>
+		<Force x="-472.37653470446924" y="177.60717171147468" z="-598.2337795367313"/>
+		<Force x="-831.8656613762369" y="-697.3968704143962" z="118.27351926638522"/>
+		<Force x="145.50516387341443" y="-447.01752589853857" z="107.63036632565452"/>
+		<Force x="640.517136974137" y="-503.35776647843437" z="-.8670424795452405"/>
+		<Force x="-496.30894993397146" y="656.5160735361735" z="-8.917081218912516"/>
+		<Force x="-59.970511954986875" y="737.1778838172511" z="-502.23032550645587"/>
+		<Force x="254.31038695690637" y="-85.15306750405736" z="812.5976148041792"/>
+		<Force x="-466.6850842197429" y="33.649287989524055" z="364.8967514610765"/>
+		<Force x="-227.58922404647078" y="148.57533897443085" z="-811.9377489956292"/>
+		<Force x="839.6940799021637" y="56.97466842408522" z="-318.8521475939689"/>
+		<Force x="362.3272180197968" y="-706.8489409774945" z="-264.4258472168631"/>
+		<Force x="-505.8214032425471" y="-723.0346848237261" z="324.9594864383717"/>
+		<Force x="-568.8744161828374" y="266.5494507684997" z="-30.478132146722988"/>
+		<Force x="-485.2241615788016" y="939.5060141587863" z="114.86701913403719"/>
+		<Force x="762.0358996664957" y="-84.19661079197815" z="105.88781416394265"/>
+		<Force x="349.4477297990101" y="-1429.4764613260852" z="-5.030937078796498"/>
+		<Force x="-567.6667253076068" y="689.2675715661583" z="-.6763489861962313"/>
+		<Force x="313.27996277450796" y="454.69440232536095" z="551.5915187153109"/>
+		<Force x="313.05157212187447" y="452.19258398876815" z="-552.5454929317236"/>
```

Comment:
Here again we need to add new tests, not change the existing ones.  We need to verify that existing code still produces the same results.